libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -278,14 +278,21 @@ MAV_RESULT AP_Mount::handle_command_do_gimbal_manager_pitchyaw(const mavlink_com
         return MAV_RESULT_ACCEPTED;
     }
 
-    // To-Do: handle gimbal device id
+    // check gimbal device id.  0 or 1 is 1st gimbal, 2 is 2nd gimbal, etc
+    uint8_t gimbal_instance = 0;
+    if ((packet.param7 > 0) && (packet.param7 <= AP_MOUNT_MAX_INSTANCES)) {
+        gimbal_instance = (uint8_t)packet.param5 - 1;
```

Comment:
should be param7?

Reply:
oops fixed.  I will retest and figure out how I missed this.